helper.go

```diff
@@ -269,6 +270,19 @@ func LifecycleStart(appName string, args interface{}) {
 	Log.WithFields(fields).Infof("starting application: %v", appName)
 }
 
+// LifecycleStarted logs that an application has started.
+func LifecycleStarted(appName string) {
+	fields := logrus.Fields{}
+	fields["type"] = "lifecycle"
+	fields["event"] = "started"
+	for _, env := range LifecycleEnvVars {
+		if os.Getenv(env) != "" {
+			fields[strings.ToLower(env)] = os.Getenv(env)
+		}
+	}
```

Comment:
This is duplicated and should be moved to a function.

Reply:
Also the LifecycleStop function has a different implementation that should be streamlined